pocs/utils/rs232.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
                  port=None,
                  baudrate=115200,
```

Comment:
I think we should remove the baudrate, and require that it be provided in the config.

Reply:
Maybe a more generic PR for #181 to do this? I could do here though.

Reply:
OK, a more generic PR addressing #181 is fine. I'd be happy to work on that when you're done with this.